packages/web-react/src/components/Toast/useToast.ts

```diff
@@ -0,0 +1,12 @@
+import { useContext } from 'react';
+import { ToastContext } from './ToastContext';
+
+export const useToast = () => {
+  const currentToastContext = useContext(ToastContext);
+
+  if (!currentToastContext) {
+    throw new Error('useToast has to be used within <useToast>');
```

Comment:
Did you mean `ToastProvider`?

```suggestion
    throw new Error('useToast has to be used within <ToastProvider>');
```

Reply:
yes, my mistake, thank you!